src/components/daily/parametertooltip/ParameterTooltip.js

```diff
@@ -0,0 +1,116 @@
+/*
+ * == BSD2 LICENSE ==
+ * Copyright (c) 2016, Tidepool Project
+ *
+ * This program is free software; you can redistribute it and/or modify it under
+ * the terms of the associated License, which is identical to the BSD 2-Clause
+ * License as published by the Open Source Initiative at opensource.org.
+ *
+ * This program is distributed in the hope that it will be useful, but WITHOUT
+ * ANY WARRANTY; without even the implied warranty of MERCHANTABILITY or FITNESS
+ * FOR A PARTICULAR PURPOSE. See the License for more details.
+ *
+ * You should have received a copy of the License along with this program; if
+ * not, you can obtain one from Tidepool Project at tidepool.org.
+ * == BSD2 LICENSE ==
+ */
+
+import React, { PropTypes, PureComponent } from 'react';
+
+import { formatLocalizedFromUTC, getHourMinuteFormat } from '../../../utils/datetime';
+
+import Tooltip from '../../common/tooltips/Tooltip';
+import colors from '../../../styles/colors.css';
+import styles from './ParameterTooltip.css';
+import i18next from 'i18next';
+
+const t = i18next.t.bind(i18next);
+
+class ParameterTooltip extends PureComponent {
```

Comment:
```suggestion
class ParameterTooltip extends React.Component {
```
This is not a PureComponent, let's use the standard Component

Reply:
same as above https://diabeloop.atlassian.net/browse/PT-1146